.github/workflows/release-tag.yml

```diff
@@ -133,3 +130,36 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
         SIGN_KEY_NAME: ${{ secrets.SIGN_KEY_NAME }}
+    - name: Determine if released version is latest
+      id: latest-version
+      run: |
+        test "$(git tag | grep -P '^v3.\d+\.\d+$' | sort -V | tail -n 1)" != '${{ github.ref_name }}'
+        echo "::set-output name=latest::$?"
+    - name: Tag latest
+      if: steps.latest-version.outputs.latest == '1'
+      run: |
+        export VERSION=$(echo ${{ github.ref_name }} | grep -Po '3\.\d+')
+        docker tag ${DOCKER_PREFIX}/lorawan-stack:${VERSION}-amd64 ${DOCKER_PREFIX}/lorawan-stack:latest-amd64
+        docker tag ${DOCKER_PREFIX}/lorawan-stack:${VERSION}-arm64 ${DOCKER_PREFIX}/lorawan-stack:latest-arm64
+        docker tag ${DOCKER_PREFIX}/lorawan-stack:${VERSION}-armv7 ${DOCKER_PREFIX}/lorawan-stack:latest-armv7
+        docker push ${DOCKER_PREFIX}/lorawan-stack:latest-amd64
+        docker push ${DOCKER_PREFIX}/lorawan-stack:latest-arm64
+        docker push ${DOCKER_PREFIX}/lorawan-stack:latest-armv7
+        docker manifest create ${DOCKER_PREFIX}/lorawan-stack:latest ${DOCKER_PREFIX}/lorawan-stack:latest-amd64 ${DOCKER_PREFIX}/lorawan-stack:latest-arm64 ${DOCKER_PREFIX}/lorawan-stack:latest-armv7
+        docker manifest push ${DOCKER_PREFIX}/lorawan-stack:latest
+      env:
+        DOCKER_CLI_EXPERIMENTAL: enabled
+        DOCKER_PREFIX: thethingsnetwork
+    - name: Prepare merge PR
+      run: |
+        gh pr create \
+            --assignee '${{ github.repository }}/stack-maintainers' \
```

Comment:
Have you tested this? Because as far as I know, you can't assign PRs to teams (yet; would be nice to have).

Reply:
No. First I want to go through all the review rounds, and at the end test. It doesn't make sense to go through entire workflow after every single comment

Reply:
FTR my comments have been dealt with, so feel free to test this.